integration_tests/cli/modelinfo-demo.sh

```diff
@@ -25,17 +25,17 @@ create_new_account vendor_account "Vendor"
 
 vid=$RANDOM
 pid=$RANDOM
-name="Device #1"
+productName="Device #1"
 echo "Add Model with VID: $vid PID: $pid"
-result=$(echo "test1234" | dclcli tx modelinfo add-model --vid=$vid --pid=$pid --name="$name" --description="Device Description" --sku="SKU12FS" --hardware-version="1.1" --firmware-version="2.0" --tis-or-trp-testing-completed=false --from $vendor_account --yes)
+result=$(echo "test1234" | dclcli tx modelinfo add-model --vid=$vid --pid=$pid --supportURL="https://originalsupporturl.test" --productName="$productName" --description="Device Description" --sku="SKU12FS" --softwareVersion="10123" --softwareVersionString="1.0b123"  --hardwareVersion="5123" --hardwareVersionString="5.1.23"  --cdVersionNumber="32" --from $vendor_account --yes)
```

Comment:
Can we have an integration test where we create a model with all possible fields ?